libtiledbsoma/src/reindexer/reindexer.cc

```diff
@@ -48,9 +48,19 @@ namespace tiledbsoma {
 
 void IntIndexer::map_locations(const int64_t* keys, int size, int threads) {
```

Comment:
Why are `threads` and `size` params signed int?  I note that ThreadPool accepts a `size_t`, so there is already a cast occurring....

Using appropriate types seems like a better solution, unless there is some sort of upstream requirement for signed values.

Reply:
Im setting both of them to size_t